aws/required-permissions.md

```diff
@@ -69,6 +69,7 @@ Check out the templates directly for more information.
 | `logs:StopQuery` | Action | Allow Cloud One File Storage Security to stop a log query of the Lambda functions created by Cloud One File Storage Security. |
 | `logs:GetQueryResults` | Action | Allow Cloud One File Storage Security to get the log query results of the Lambda functions created by Cloud One File Storage Security. |
 | `logs:FilterLogEvents` | Action | Allow Cloud One File Storage Security to filter the log events of the Lambda functions created by Cloud One File Storage Security. |
+| `logs:TagResource` | Action | Allow Cloud One File Storage Security to tag CloudWatch log resource. |
```

Comment:
There's no role/policy that has this action in the scanner or storage stack template, so we don't need to put it here.

Reply:
Removed